ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class PostCommand extends IronfishCommand {
+  static description = `Post a raw transaction`
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  static args = [
+    {
+      name: 'transaction',
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      description: 'The raw transaction in hex encoding',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { args } = await this.parse(PostCommand)
+    const transaction = args.transaction as string
+
+    CliUx.ux.action.start(`Posting transaction`)
+    const client = await this.sdk.connectRpc()
+    const response = await client.postTransaction({ transaction })
+    CliUx.ux.action.stop()
```

Comment:
Can you display the transaction returned in response? User needs this to broadcast transaction

Reply:
It's already happening below here, look a few lines down.

Reply:
I mean display the details in transaction like what send cli does. 
```
Sending ${CurrencyUtils.renderIron(amount, true, assetId)} to ${to} from ${from}
Transaction Hash: ${transaction.hash().toString('hex')}
Transaction fee: ${CurrencyUtils.renderIron(transaction.fee(), true)}
```
also add description, user can use this result and call broadcast cli to post on network like what bitcoin dose.

Reply:
The fee is already in the confirmation, but I'll add the hash.

The CLI command shouldn't output tutorial steps. That's added to the docs / command CLI help like you asked for.

Reply:
I added the posted hash to the message.